### PR TITLE
Add File mounts to play kube

### PR DESCRIPTION
**Pull request by haircommander**
Both File and FileOrCreate options are supported.

Signed-off-by: Peter Hunt <pehunt@redhat.com>

### CODE REVIEW

**Comment by haircommander** [comment]
Fixes: #2922

File: cmd/podman/play_kube.go [hunk 1]
```diff
@@ -169,7 +171,19 @@ func playKubeYAMLCmd(c *cliconfig.KubePlayValues) error {
 					return errors.Wrapf(err, "Error giving %s a label", hostPath.Path)
 				}
 				break
+			case v1.HostPathFileOrCreate:
+				if _, err := os.Stat(hostPath.Path); os.IsNotExist(err) {
+					if _, err := os.OpenFile(hostPath.Path, os.O_RDONLY|os.O_CREATE, createFilePermission); err != nil {
```
Comment (vrothberg) [inline cmd/podman/play_kube.go]: I think we need to close the file to avoid leaking the fd.

**Comment by rhatdan** [comment]
LGTM other then closing the FD.

File: cmd/podman/play_kube.go (hunk 1 above)
Comment (haircommander) [reply cmd/podman/play_kube.go]: fixed!

**Comment by TomSweeneyRedHat** [comment]
LGTM
Was wondering if we should defer the f.close, but I think it's good as you have it.

**Comment by haircommander** [comment]
considering we don't need the file* after the file is created, I don't think a defer is really worth it. mine as well clean up asap :)

**Pull request merged**